src/workspaces/list/WorkspaceStateCell.tsx

```diff
@@ -0,0 +1,149 @@
+import { icon, IconId, Modal } from '@terra-ui-packages/components';
+import React, { ReactNode, useState } from 'react';
+import { Link } from 'src/components/common';
+import ErrorView from 'src/components/ErrorView';
+import { FirstParagraphMarkdownViewer } from 'src/components/markdown';
+import colors from 'src/libs/colors';
+import * as Style from 'src/libs/style';
+import { WorkspaceWrapper as Workspace } from 'src/workspaces/utils';
+
+const styles = {
+  tableCellContainer: {
+    height: '100%',
+    padding: '0.5rem 0',
+    paddingRight: '2rem',
+    borderTop: `1px solid ${colors.light()}`,
+  },
+  tableCellContent: {
+    height: '50%',
+    display: 'flex',
+    alignItems: 'center',
+  },
+};
+
+const WorkspaceDescriptionCell = (props: { description: unknown | undefined }) => (
+  <div style={styles.tableCellContent}>
+    <FirstParagraphMarkdownViewer
+      style={{
+        height: '1.5rem',
+        margin: 0,
+        ...Style.noWrapEllipsis,
+        color: props.description ? undefined : colors.dark(0.75),
+        fontSize: 14,
+      }}
+      renderers={{}} // needed to make typechecker work, because FirstParagraphMarkdownViewer is not typed
```

Comment:
I suppose I could alse have just added a `// @ts-expect-error` instead.

Reply:
Generally, suggest `ts-expect-error` for cases like this so that workarounds can be easily cleaned up when the relevant component (`FirstParagraphMarkdownViewer` in this case) is converted to TypeScript.